app/Services/User/UserUpdateService.php

```
@@ -6,12 +6,49 @@
 
 class UserUpdateService
 {
+    /**
+     * Create a new user with the given validated attributes, role, and
+     * department. Matches the previous inline UsersController@store logic
+     * exactly - including the language fallback to 'en' for anything
+     * outside the 3 supported locales.
+     */
+    public function create(array $validated, ?UploadedFile $imageFile): User
+    {
+        $settings = Setting::cached();
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Models\Setting' in method 'create'. (undefined)

(StaticAccess)